Common/Converter.php

```diff
@@ -33,16 +33,11 @@ public static function toAscii(string $str): string
         return $clean;
     }
 
-    public static function formatBytes(int $bytes, int $precision = 2): string
+    public static function formatBytes($size, $precision = 2)
```

Comment:
Hello, you refactored this function, before your unit test was written. 
To make sure that this refactor is sound, you should first write a test. commit it, then do the refactor. That way we can easily test if the refactoring is working correctly.